static/src/js/containers/PortalLinkContainer/PortalLinkContainer.jsx

```diff
@@ -102,11 +101,27 @@ export const PortalLinkContainer = (props) => {
   }
 
   if (type === 'button') {
+    // We need all the props passed in to PortalLinkContainer to be passed into the Button component.
+    // This is taking all the props we destructed and putting them into an object to spread on Button
+    const allProps = {
```

Comment:
I feel it might only be marginally better if it is at all, but I'd kinda prefer to see these props just passed by name as attributes on Button with a `{...otherProps}` at the end instead of creating this temporary object.